data/data/agent/files/usr/local/bin/add-node.sh

```diff
@@ -10,7 +10,7 @@ cluster_id=""
 while [[ "${cluster_id}" = "" ]]
 do
     # Get cluster id
-    cluster_id=$(curl -s -S "${BASE_URL}/clusters" | jq -r .[].id)
+    cluster_id=$(curl -s -S "${BASE_URL}/clusters" -H "Authorization: ${AGENT_AUTH_TOKEN}" | jq -r .[].id)
```

Comment:
This approach could be potentially error prone, given the high number of scripts where the curl + token auth is invoked. Even though on the longer term IMHO we should migrate all the bash code into the cli tool, for now it could be enough to define a utility function (`curl_assisted_service` ?) that embeds this piece of config, requiring to the user the min amount of info

Reply:
Yes, sure